### PR TITLE
alfis: 0.6.5 -> 0.6.9

### PR DESCRIPTION
**Pull request by misuzu**
###### Motivation for this change
https://github.com/Revertron/Alfis/releases/tag/v0.6.6
https://github.com/Revertron/Alfis/releases/tag/v0.6.7
https://github.com/Revertron/Alfis/releases/tag/v0.6.8
https://github.com/Revertron/Alfis/releases/tag/v0.6.9
https://github.com/Revertron/Alfis/compare/v0.6.5...v0.6.9

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [X] aarch64-linux
  - [X] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by misuzu** [comment]
@ofborg build alfis alfis-nogui

**Comment by hugolgst** [comment]
Result of `nixpkgs-review pr 138103` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>alfis</li>
    <li>alfis-nogui</li>
  </ul>
</details>

**Comment by misuzu** [comment]
@ofborg build alfis alfis-nogui

**Comment by hugolgst** [comment]
Result of `nixpkgs-review pr 138103` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>alfis</li>
    <li>alfis-nogui</li>
  </ul>
</details>

**Comment by misuzu** [comment]
> Failing again unfortunately
> 
> Result of `nixpkgs-review pr 138103` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
> 
> 2 packages failed to build:

It builds fine on [ofborg](https://github.com/NixOS/nixpkgs/pull/138103/checks?check_run_id=3673135013). Do you have any logs?

**Comment by misuzu** [comment]
Finally, got my hands on Apple hardware.
@ofborg build alfis alfis-nogui

**Comment by misuzu** [comment]
Result of `nixpkgs-review pr 138103` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>alfis</li>
    <li>alfis-nogui</li>
  </ul>
</details>

File: pkgs/applications/blockchains/alfis/default.nix [hunk 1]
```diff
@@ -1,25 +1,29 @@
 { stdenv, lib, fetchFromGitHub, rustPlatform, pkg-config
+, Security
 , withGui ? true, webkitgtk, Cocoa, WebKit
 }:
-
+let
+  cargoFlags = [
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/alfis/default.nix:6]: Alternative to that would be doing ``cargoTestFlags = cargoBuildFlags;``.

**Comment by SuperSandro2000** [comment]
but more importantly we need to resolve the merge conflict.

**Comment by misuzu** [comment]
> but more importantly we need to resolve the merge conflict.

Rebased.

**Comment by misuzu** [comment]
@ofborg build alfis alfis-nogui

**Comment by misuzu** [comment]
@ofborg build alfis alfis-nogui

**Pull request merged**